exercises/exercises.csproj

```diff
@@ -38,7 +38,7 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="sgf-parsing\SgfParsing.cs" />
+    <Compile Include="**\*.cs" Exclude="sgf-parsing\SgfParsing.cs;markdown\Markdown.cs" />
```

Comment:
@ErikSchierboom - can you see this comment? I just "published" the review so I'm not sure it was visible before that (I'm still learning the nuances of GitHub)